### ISSUE TITLE
Consider ignoring coinbase transaction for block audit percentages

### ISSUE THREAD

**Issue by nymkappa** [mempool/mempool#4249]
### Description

To me it would make more sense to display -100% for all values on empty blocks:
- We don't know in advance what the coinbase transaction will be, therefore it does not make sense to include it in the comparison at all
- A block must at least contains the coinbase transaction to be valid, giving another reason to ignore it from the audit since it must be there all the time

@mononaut what do you think?

### Context

https://mempool.space/block/000000000000000000001c2d398b7d20e9ab5a374019e8ff339d14dcb1770c29

<img width="1086" alt="image" src="https://github.com/mempool/mempool/assets/9780671/c65b7276-61e9-4123-b3b0-772a1dd8685e">

**Comment by mononaut** [comment]
yeah, we already exclude the coinbase from the health score calculation and audit highlighting, so maybe it does make sense to also ignore it for those projected vs actual summary fields.

my only concern would be that then they wouldn't match the weight/transaction count displayed elsewhere on the page:
<img width="325" alt="Screenshot 2023-09-12 at 7 56 51 PM" src="https://github.com/mempool/mempool/assets/83316221/9fdcc092-cf2b-44df-8c1d-1c2d444e71b9">
<img width="361" alt="Screenshot 2023-09-12 at 7 57 04 PM" src="https://github.com/mempool/mempool/assets/83316221/bec80e44-297a-4f42-a87c-81aeb8cbd4b0">
